### ISSUE TITLE
Latest version causes immediate safe mode crash.

### ISSUE THREAD

**Issue by Tweaker177** [NovelCOVID/API#557]
I had the original version for a long time working great, updated to the first April fools version and that one worked, got the most recent update and it causes an immediate crash. I tried downgrading in Cydia but the last version isn’t showing the correct numbers (it is negative at the moment for USA total cases setting).

Here’s the crash report....
https://pastebin.com/PhHXg748

Thanks!

**Comment by ebwinters** [comment]
I don't even know what you're talking about. cydia is an ios app, we provide an API service and a node wrapper for the service. Closing for now feel free to keep thread going though

**Comment by puf17640** [comment]
lmao

**Comment by Tweaker177** [comment]
Sorry Cydia doesn’t have anything to do with it, besides being the package manager used to download an extension that listed this API as the source code. I didn’t look at the code or full Readme, I just followed the link in the settings app listing this as the source code. (Not realizing this was just the API’s source, not the extension’s.)

Obviously crashing iOS’s SpringBoard  isn’t going to be from your API, but from the implementation of the tweak. Sorry I listed the issue here. The incorrect number of cases displaying though could have very well been from this API, as it worked before and I doubt the extension changed how it uses your code. So sorry for accidentally reporting to the wrong developer(s). Your API may still have an issue though.

Glad you got a laugh about something you’re clueless about, and ignored the part of the issue that was relevant to your API.
